tools/challenge-editor/client/src/components/editor/editor.tsx

```diff
@@ -87,6 +88,15 @@ const Editor = () => {
       <p>
         <Link to={`/${superblock || ''}/${block || ''}`}>Return to Block</Link>
       </p>
+      <p>
+        <Link
+          to={`${import.meta.env.CHALLENGE_EDITOR_LEARN_CLIENT_LOCATION || 'http://localhost:8000'}/learn/${superBlockNameMap[superblock || '']}/${block || ''}/${items.name.replace(/[\s]+/g, '-').toLowerCase() || ''}`}
```

Comment:
```suggestion
          to={`${import.meta.env.CHALLENGE_EDITOR_LEARN_CLIENT_LOCATION}/learn/${superBlockNameMap[superblock || '']}/${block || ''}/${items.name.replace(/[\s]+/g, '-').toLowerCase() || ''}`}
```
Nit: we don't need to provide the same default twice.